src/RegistrationManager.ts

```diff
@@ -210,14 +210,14 @@ export class RegistrationManager {
 
     async getRelayData(): Promise<RelayManagerData> {
         const relayData: RelayManagerData[] =
-            await this.contractInteractor.getRelayInfo([this.managerAddress]);
+            await this.contractInteractor.getRelayInfo(new Set<string>(this.managerAddress));
```

Comment:
The problem here is that the `Set` constructor receives an iterable and it builds the set starting from that iterable. The string object is iterable itself (an iterable of the chars which the object is composed by), so it will create a set of characters, while we need to create a `Set` with a single item. 
We need to use an array with a single element as input argument for the constructor. 
```suggestion
            await this.contractInteractor.getRelayInfo(new Set<string>([this.managerAddress]));
```